src/test/java/net/snowflake/ingest/streaming/internal/it/BigFilesITBase.java

```diff
@@ -35,16 +38,15 @@ public class StreamingIngestBigFilesIT {
   private SnowflakeStreamingIngestClientInternal<?> client;
   private Connection jdbcConnection;
   private String testDb;
+  private boolean enableIcebergStreaming;
 
-  @Parameters(name = "{index}: {0}")
-  public static Object[] compressionAlgorithms() {
-    return new Object[] {"GZIP", "ZSTD"};
-  }
+  @Parameter(0)
+  public String compressionAlgorithm;
 
-  @Parameter public String compressionAlgorithm;
+  @Parameter(1)
+  public Constants.IcebergSerializationPolicy icebergSerializationPolicy;
```

Comment:
lets remove the runwith parameterized from the base class, and remove the parameter annotations from here too. Makes class design murky.
Let the derived classes have the runwith and the @Parameter, and pass in the values via beforeAll.

I'm also debating the value of running this IT for iceberg when you've already authored a much better large file test thats in my PR rn.

Reply:
clearly its not exercising multipart upload, if it was iceberg tests would've failed.

Reply:
Removed this IT and use the on with larger file size for Iceberg streaming.